documentation/puppet-api/v3/environment_modules.markdown

```diff
@@ -16,10 +16,55 @@ on Ruby Puppet masters, such as the [deprecated WEBrick Puppet master][]. It als
 the Ruby-based Puppet master's authorization methods and configuration. See the
 [Authorization section](#authorization) for more information.
 
-## `GET /puppet/v3/environment_modules?environment=:environment`
+## `GET /puppet/v3/environment_modules`
```

Comment:
I assume we'll also need to make some updates to the `environment_classes.json` schema to allow for the module info either to be for a single environment entry vs. a list of environment entries?